### DISCUSSION TITLE
Is it possible that this could be used in another engine?

### DISCUSSION THREAD

**Discussion by catweaselirl** [Joshua-Ashton/VPhysics-Jolt#149] [Q&A] [answered]
Sorry for the dumb question. Just curious. Thanks :)

Edit: just be clear, I know it won't work "out of the box" in another engine, I was moreso just curious if it could be "easily" ported and used in another engine, or if that would basically be writing it from scratch for the new engine.

**Comment by misyltoad** [comment, marked as answer]
https://github.com/jrouwe/JoltPhysics/

**Reply by catweaselirl** [reply]
Omg I literally found this out when I first heard of this project a few weeks ago, and somehow forgot when I remembered about it today. Thanks!
